data/json/monstergroups.json

```diff
@@ -2095,6 +2095,7 @@
       { "monster": "mon_skeleton_hulk", "freq": 1, "cost_multiplier": 50 },
       { "monster": "mon_zombie_master", "freq": 1, "cost_multiplier": 30 },
       { "monster": "mon_beekeeper", "freq": 1, "cost_multiplier": 5 },
+      { "monster": "mon_zombie_thorny", "freq" : 5, "cost_multiplier" : 25 },
```

Comment:
I don't recommend adding new types of zombies into GROUP_ZOMBIE, as this is really supposed to be an early game list that should be revised to have fewer of this sort of critter.

Reply:
I see. 
Less special specials for the early game, eh?

Reply:
Yeah, in lore terms it's mostly that they need some time to mutate.

Reply:
How about butting this into:
GROUP_ZOMBIE_UPGRADE

{ "monster": "mon_zombie_thorny", "freq": 9, "cost_multiplier": 5 },

Reply:
This way it will evolve froum usual zombies with relatively small chance.

Reply:
Them being more common when upgraded was intended, but if they also need to be in `GROUP_ZOMBIE_UPGRADE` for that to work then so be it.